docs/infrastructure/deployment-targets/azure/web-app-targets/index.md

```diff
@@ -6,12 +6,26 @@ position: 20
 
 Azure Web App deployment targets allow you to reference existing Web Apps in your Azure subscription, that you can then reference by role during deployments.
 
+:::hint
+From version 2022.1 Octopus can discover Azure Web App targets using tags on your Web App resource.
+:::
+
 ## Requirements
 
 - You need an [Azure Service Principal account](/docs/infrastructure/accounts/azure/index.md#azure-service-principal) that references your Azure subscription.
 
 - Once your Azure account is setup, you need an existing Azure Web App / App Service setup within your Azure subscription. To learn more about App Services, see the [Azure App Services documentation](https://docs.microsoft.com/en-us/azure/app-service/) that can help you get started. If you are dynamically creating the web app during your deployment, check our section about [creating Web App targets by scripts using service messages](#creating-web-app-targets-by-scripts).
 
+## Discovering web app targets
+
+Octopus can discover Azure Web App targets as part of your deployment using tags on your resource. To discover targets use the following steps.
+
+- Add an Azure account variable **Octopus.Azure.Account** to your project.
+- Add tags to your Azure Web App so that Octopus can match it to your deployment step and environment.
```

Comment:
I feel like we should link the "Add tags" words here to the relevant doco, otherwise it feels like an immediate questionmark